H5P/laravel-h5p/src/LaravelH5p/Http/Controllers/AjaxController.php

```diff
@@ -199,8 +310,29 @@ public function finish(Request $request)
         exit;
     }
 
+    /**
+     * H5P content user data 
+     *
+     * Save H5P content user data
+     *
+     * @bodyParam content_id int required H5P content id
+     * @bodyParam data_type string required Example H5P-Blanks-question-important-description-open
+     * @bodyParam sub_content_id int required Example 0
+     * 
+     * @response {
+     *   "success": [
+     *     "true"
+     *   ]
```

Comment:
Response is not correct. It is not array of true. It is just boolean true.

Reply:
corrected